ldap/servers/slapd/back-ldbm/db-mdb/mdb_debug.c

```diff
@@ -533,7 +549,7 @@ void dbmdb_log_dbi_set_fn(const char *file, int lineno, const char *funcname, co
     Dl_info info = {0};
     dladdr(fn, &info);
     /* Cannot use dbi_str here because slot is not yet up2date (so dbname is an argument) */
-    dbg_log(file, lineno, funcname, DBGMDB_LEVEL_MDBAPI,  "%s(txn=0x%p, dbi=%d <%s>, fn=0x%p <%s>)\n", action, txn, dbi, dbname, fn, info.dli_sname);
+    dbg_log(file, lineno, funcname, DBGMDB_LEVEL_MDBAPI, "%s(txn=0x%p, dbi=%d <%s>, fn=0x%p <%s>)\n", action, txn, dbi, dbname, fn, info.dli_sname);
```

Comment:
Why not calling txn_loglvl() instead of DBGMDB_LEVEL_MDBAPI?

Reply:
txn_log() is used only for txn  related function It allows to log when txn are open and closed when 
 DBGMDB_LEVEL_MDBAPI or DBGMDB_LEVEL_TXN is set but log txn backtrace only if DBGMDB_LEVEL_TXN is set